packages/openapi-generator/src/options/flags.ts

```diff
@@ -0,0 +1,101 @@
+    description:
+      'Allow to overwrite files, that already exist. This is useful, when running the generation regularly.',
+    default: false
+  }),
+  clearOutputDir: flags.boolean({
+    description:
+      'Remove all files in the output directory before generation. Be cautious when using this option, as it really removes EVERYTHING in the output directory.',
+    default: false
+  }),
+  skipValidation: flags.boolean({
+    description:
+      'By default, the generation fails, when there are duplicate or invalid names for operations and/or path parameters after transforming them to camel case. Set this to true to enable unique and valid name generation. The names will then be generated by appending numbers and prepending prefixes.',
+    default: false
+  }),
+  tsConfig: flags.string({
+    description:
+      'Replace the default `tsconfig.json` by passing a path to a custom config. By default, a `tsconfig.json` is only generated, when transpilation is enabled (`--transpile`). If a directory is passed, a `tsconfig.json` file is read from this directory.',
+    parse: input => resolve(input),
+    helpValue: '<path/to/tsconfig.json>'
+  }),
+  packageJson: flags.boolean({
+    description:
+      'When enabled, a `package.json`, that specifies dependencies and scripts for transpilation and documentation generation is generated.',
+    default: false
+  }),
+  verbose: flags.boolean({
+    description: 'Turn on verbose logging.',
+    default: false
+  }),
+  optionsPerService: flags.string({
+    description:
+      'Set the path to a file containing the options per service. The configuration allows to set a `directoryName` and `packageName` for every service, identified by the path to the original file. It also makes sure that names do not change between generator runs. If a directory is passed, a `options-per-service.json` file is read/created in this directory.',
+    helpValue: '<path/to/options-per-service.json>',
+    parse: input => {
+      const isFilePath =
+        (existsSync(input) && lstatSync(input).isFile()) || !!extname(input);
+      return isFilePath
+        ? resolve(input)
+        : resolve(input, 'options-per-service.json');
+    }
+  }),
+  packageVersion: flags.string({
+    description: 'Set the version in the generated package.json.',
+    default: '1.0.0',
+    hidden: true
+  }),
+  readme: flags.boolean({
+    description:
+      'Generate default `README.md` files in the client directories.',
+    default: false,
+    hidden: true
+  }),
+  metadata: flags.boolean({
+    description: 'When enabled, metadata for the API hub is generated.',
+    default: false,
+    hidden: true
+  }),
+  config: flags.string({
+    char: 'c',
+    parse: input => resolve(input),
+    description:
+      'Set the path to the file containing the options for generation. If other flags are used, they overwrite the options set in the config. If a directory is passed, a `config.json` file is read from this directory.',
```

Comment:
```suggestion
      'Set the path to a file containing the options for generation instead of setting the options on the command line. When combining the `config` option with other options on the command line, the command line options take precedence. If a directory is passed, a `config.json` file is read from this directory.',
```